src/test/jtx/Env.h

```diff
@@ -152,9 +145,40 @@ all_features_except (Col const& keys)
  */
 template <class... Args>
 FeatureBitset
-all_features_except (uint256 const& key, Args const&... args)
+supported_features_except (uint256 const& key, Args const&... args)
+{
+    return supported_features_except(
+        std::array<uint256, 1 + sizeof...(args)>{{key, args...}});
+}
+
+/**
+ * @brief create collection of features to pass to Env ctor
+ *
+ * The resulting collection will contain *all supported amendments* plus
+ * the features passed as arguments.
+ *
+ * @param keys features to exclude from the resulting collection
+ */
+template<class Col>
+FeatureBitset
+supported_features_plus (Col const& keys)
```

Comment:
See my comment in the other PR for names, but I don't like all these functions of the form `primitive1_and_primitive2(){primitive1(); primitive2();}`.

I have some ideas for how to refactor this, we don't need to do it in this PR. Just a placeholder comment.